docs-kits/kits/Quality-Kit/page_adoption-view.md

```diff
@@ -421,22 +239,23 @@ To realize the **Business Logic** described in the Quality Kit
 
 all steps of the **Business Process** (described in the [Development View](https://confluence.catena-x.net/display/BDPQ/WORK+IN+PROGRESS+-+Quality+KIT+-+Development+View)), like data provisioning and consuming by the involved partner companies, are realized in compliance with the [Catena-X Data Governance Framework](https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations).  Under this link you can find the latest version of the framework regulations as download. The documents are seperated in the following levels:
 
-![quality kit data offering diagram](@site/static/img/quality-kit_data-offering-min.png)
-
-<!-- BROKEN LINK............... -->
 **Data Space Level**: 10 Golden Rules of Catena-X
 
-**Use Case Level**: Quality Management specific policy as download: <a target="_blank" href={ require("/office/20230710_Catena-X_UseCasePolicy_Quality_3.0_EN.docx").default } download>20230710_Catena-X_UseCasePolicy_Quality_3.0_EN.docx</a>
+
+**Use Case Level**: [Quality Management specific policy](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_Quality.pdf)![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/148753489/3d45270a-3045-41b7-a187-8ce9789f497c)
```

Comment:
Hi @ClaudiaKilian references to restricted content for Catena-X should be not present on Eclipste Tractus-X Webpage

Reply:
Hi @FaGru3n, since @ClaudiaKilian is absent... I'm not sure what is the concrete issue here. The links to specific files? But how should a reference to a standard should be realized. The files we reference to are stored in a public area.

Reply:
Hi @ChristophMarquardt thanks for the information. Guessed that you need specific permission to view theese kind of files.

and looks suspicious if linking catena related information into our Eclipse Webpage.

Reply:
Hi @FaGru3n, ok, then we keep it like it is, right?

Reply:
Hi Fabian @FaGru3n  
the links go to public content of Catena-X. We added them not to have double copied content. So it is always up to date and we have less maintenance effort.
Ok for you?

Reply:
Just want to make sure that no content is copied from Catena-X restricted permission level